docs/use-cases/navigate.md

```diff
@@ -1,8 +1,54 @@
 ---
-title: "Teleoperate and navigate rovers"
+title: "Teleoperate and navigate with rovers"
```

Comment:
I think teleoperate is a transitive verb and navigate is used with the transitive meaning - you're navigating the rover (with your Viam-powered software), so I think this was correct as is. What is your thinking?

Reply:
Oh that does sound correct, sorry! Fixing